src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PerformTouchAction.java

```diff
@@ -0,0 +1,360 @@
+		}
+		js.executeScript("window.scrollTo(0, 0)");
+		appWebviewHeight = Integer.parseInt(js.executeScript("return $(window).height()").toString());
+		appWebviewWidth = Integer.parseInt(js.executeScript("return $(window).width()").toString());
+		loadedPageHeight = Integer.parseInt(js.executeScript("return $(document).height()").toString());
+		loadedPageWidth = Integer.parseInt(js.executeScript("return $(document).width()").toString());
+		ratio = nativeWidth / appWebviewWidth;
+		webviewHeight = nativeHeight / ratio;
+		webviewWidth = appWebviewWidth;
+		taskbarWebviewHeight = webviewHeight - appWebviewHeight;
+		taskbarWebviewWidth = webviewWidth;
+		taskbarNativeHeight = taskbarWebviewHeight * ratio;
+		taskbarNativeWidth = nativeWidth;
+		appNativeWidth = nativeWidth;
+		appNativeHeight = nativeHeight - taskbarNativeHeight;
+		varStatus();
+	}
+	
+	/**
+	 * That method return value of all resolution variables.
+	 */
+	public void varStatus() {
+		System.out.println("=======================================");
+		System.out.println("nativeHeight: " + nativeHeight); //1920
+		System.out.println("nativeWidth: " + nativeWidth); //1080
+		System.out.println("webviewHeight: " + webviewHeight); //640
+		System.out.println("webviewWidth: " + webviewWidth); //360
+		System.out.println("ratio (nativeWidth / appWebviewWidth): " + ratio); //3
+		System.out.println("=======================================");
+		System.out.println("taskbarNativeHeight: " + taskbarNativeHeight); //243 should be 75
+		System.out.println("taskbarNativeWidth: " + taskbarNativeWidth); //1080
+		System.out.println("taskbarWebviewHeight: " + taskbarWebviewHeight); //81 should be 25
+		System.out.println("taskbarWebviewWidth: " + taskbarWebviewWidth); //360
+		System.out.println("=======================================");
+		System.out.println("appNativeWidth: " + appNativeWidth); //1080
+		System.out.println("appNativeHeight: " + appNativeHeight); //1677 should be 1845
+		System.out.println("appWebviewHeight: " + appWebviewHeight); //559 should be 615
+		System.out.println("appWebviewWidth: " + appWebviewWidth); //360
+		System.out.println("=======================================");
+		System.out.println("loadedPageHeight: " + loadedPageHeight); //2484
+		System.out.println("loadedPageWidth: " + loadedPageWidth); //360
+		System.out.println("=======================================");
+	}
+	
+	/**
+	 * 
+	 * @param direction Use public const DIRECTION from that class
+	 * @param pixelPath	From 0 to half of screen X or Y, if path will be too high it will be set to edge of app
+	 * @param duration In milliseconds
+	 * @param waitAfter In milliseconds, Reccomend 2*duration
+	 */
+	public void swipeFromCenterToDirection(String direction, int pixelPath, int duration, int waitAfter){
+		int centerX = appNativeWidth / 2;
+		int centerY = (appNativeHeight / 2) + taskbarNativeHeight;
+		int path = 0;
+		if (mobileDriver.getContext() != "NATIVE_APP") {
```

Comment:
if doing a String comparison, it needs to be used with .equals()
Right now, this comparison would always hold true and trigger the next statement.